test_harvesting_arxiv_workflow_rejected fails

Current builds are failing:

https://travis-ci.org/inspirehep/inspire-next/jobs/133365652#L2127

```python
>       from invenio_workflows_ui.search import default_query_factory
E       ImportError: cannot import name default_query_factory
```

@jalavik

--- a/inspirehep/modules/workflows/tasks/matching.py
+++ b/inspirehep/modules/workflows/tasks/matching.py
@@ -226,7 +226,6 @@ def exists_in_holding_pen(obj, eng):
     from elasticsearch_dsl import Q
     from invenio_search import RecordsSearch
     from invenio_workflows_ui.utils import obj_or_import_string
-    from invenio_workflows_ui.search import default_query_factory
 
     config = current_app.config['WORKFLOWS_UI_REST_ENDPOINT']
     index = config.get('search_index')

--- a/inspirehep/utils/search.py
+++ b/inspirehep/utils/search.py
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of INSPIRE.
-# Copyright (C) 2014, 2015 CERN.
+# Copyright (C) 2016 CERN.
 #
 # INSPIRE is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
@@ -22,16 +22,17 @@
 
 """Functions for searching ES and returning the results."""
 
+from invenio_search.api import RecordsSearch
+from inspirehep.modules.search import IQ
 
-def perform_es_search(q, index, start=0, size=10, sort=None):
-    """."""
-    from invenio_search.api import RecordsSearch
-    from inspirehep.modules.search import IQ
+
+def perform_es_search(q, index, start=0, size=10, sort=None, fields=None):
+    """Helper to use elasticsearch_dsl with Spires/Invenio syntax."""
     query = IQ(q)
 
+    search = RecordsSearch(index=index).query(query)
     if sort:
-        return RecordsSearch(index=index).query(query). \
-            sort(sort)[start:start + size].execute()
-    else:
-        return RecordsSearch(index=index).query(query)[start:start + size] \
-            .execute()
+        search = search.sort(sort)
+    if fields and isinstance(fields, list):
+        search = search.extra(_source={'include': fields})
+    return search[start:start + size].execute()

--- a/setup.py
+++ b/setup.py
@@ -30,6 +30,7 @@ from setuptools import find_packages, setup
 readme = open('README.rst').read()
 
 install_requires = [
+    'amqp>=1.4.9,<2.0',
     'Flask-Gravatar>=0.4.2',
     'HarvestingKit>=0.6.2',
     'plotextractor>=0.1.2',

--- a/tests/integration/workflows/test_audit.py
+++ b/tests/integration/workflows/test_audit.py
@@ -39,6 +39,7 @@ def test_audit(app):
         db.session.add(user)
 
         workflows_object = WorkflowObject.create_object()
+        workflows_object.data_type = 'hep'
         workflows_object.save()
 
         db.session.commit()

